provider/online.go

```diff
@@ -28,6 +29,19 @@ type OnlineStore interface {
 	Provider
 }
 
+type ImportID string
+
+type Import interface {
+	Status() string
+	ErrorMessage() string
+}
+
+type ImportableOnlineStore interface {
+	OnlineStore
+	ImportTable(feature, variant string, valueType ValueType, source fs.Filepath) (ImportID, error)
+	GetImport(id ImportID) (Import, error)
+}
+
```

Comment:
@sdreyer, this was my solution to not making the DynamoDB provider struct public in the `provider` package. Let me know if you'd prefer that instead of this abstraction.